operator/internal/manifests/internal/config/loki-runtime-config.yaml

```
@@ -54,6 +54,23 @@ overrides:
     {{- if $spec.QueryLimits.CardinalityLimit }}
     cardinality_limit: {{ $spec.QueryLimits.CardinalityLimit }}
     {{- end }}
+    {{- with $l.Blocked }}
+    blocked_queries:
+    {{- range $blockedQuery := . }}
+    - {{ with $blockedQuery.Pattern -}}
+      pattern: '{{ . }}'
+      {{- end -}}
```

```suggestion
    - {{ with $blockedQuery.Pattern -}}
      pattern: |
        {{ . | yamlBlock "        " }}
      {{- end -}}
```

---

This also needs changes in `build.go` which I can not attach comments to, because it is not changed anymore:

```golang
func yamlBlock(in reflect.Value, indent string) string {
	inStr := in.String()
	lines := strings.Split(strings.TrimRight(inStr, "\n"), "\n")
	return strings.Join(lines, "\n"+indent)
}
```

and

```golang
	lokiRuntimeConfigYAMLTmpl = template.Must(template.New("runtime-config").Funcs(template.FuncMap{
		"yamlBlock": yamlBlock,
	}).ParseFS(lokiRuntimeConfigYAMLTmplFile, "loki-runtime-config.yaml"))
```